bin/groopm2

```diff
@@ -159,7 +159,8 @@ class parse_command_configure:
     def __init__(self, parser):
         parser.add_argument('dbname', help="name of the database being created")
         parser.add_argument('reference', help="fasta file containing bam reference sequences")
-        parser.add_argument('bamfiles', nargs='+', help="bam files to parse")
+        parser.add_argument('bamfiles', nargs='*', help="bam files to parse")
+        parser.add_argument('--cov_file', help="file with coverage information for reference sequences")
```

Comment:
Perhaps the help message for the option should describe the expected format of the coverage file? Funnily enough, a few other binning tools have a similar feature using slightly different file formats (see https://bitbucket.org/berkeleylab/metabat/wiki/Benchmark_MetaHIT "generating depth files" section).

Reply:
It uses the format of the 'jgi_summarize_bam_contig_depths' script provided with MetaBAT. This format is as follows:
contigName<\t>contigLen<\t>totalAvgDepth<\t><sample1>.bam<\t><sample1>.bam-var …

This is a bit non-trivial to explain so I think is best done in the manual. GroopM doesn't need all this information, but the benefit of using this format is that many people already run MetaBAT v2 so already have a file in this format. This script can also be run independently of MetaBAT v2. How would you like to handle this?

Reply:
Is it OK to say something about compatibility with the metabat -a option? The only reason to use the flag is if a user has or is going to run Metabat too, I would guess. We can give details in the manual. We should make it clear that it's optional and that people don't have to know what Metabat is to use GroopM.